lib/routes/spglobal/ratings.ts

```diff
@@ -0,0 +1,78 @@
+import { Route, ViewType } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import got from '@/utils/got';
+
+export const route: Route = {
+    path: '/ratings/:language?',
+    categories: ['finance', 'popular'],
```

Comment:
```suggestion
    categories: ['finance'],
```
New route can't be popular.